### DISCUSSION TITLE
SwaggerUI with docker and nginx

### DISCUSSION THREAD

**Discussion by gregd33** [bentoml/BentoML#1190] [Q&A] [answered]
I have a service running via docker-compose:

```
    my_service:
        restart : "always"
        image: myservice
        ports:
            - 5666:5000
        command:
            --workers 4
```
and nginx

```
         location /my_service/ {
            proxy_pass  http://127.0.0.1:5666/;
        }  
```
The page is blank, and in the console see: `Uncaught ReferenceError: SwaggerUIBundle is not define`

However, it works to call the predict endpoint. Is there a way to have the Swagger UI available?

**Comment by parano** [comment, marked as answer]
@yubozhao is working on a fix for this issue here: https://github.com/bentoml/BentoML/pull/1207, it should be available in the next release. Thanks for reporting it!

**Reply by gregd33** [reply]
Just wanted to confirm it does work within docker (even behind nginx)
